### PR TITLE
Added CHAIN token to the Base token list

### PR DESCRIPTION
**Pull request by kr-pratikp**
**Description**

Added details for CHAIN token to add it in the Base Goerli's token list

### CODE REVIEW

**Comment by roberto-bayardo** [comment]
Thanks for this submission. We'll get back to this after next week when the team is back in the office.

**Comment by roberto-bayardo** [comment]
We'll take a look in the coming week.

**Comment by kr-pratikp** [comment]
Deployed ERC-20(CHAIN) token contract using OptimismMintableERC20Factory. Please review it.

**Comment by roberto-bayardo** [comment]
Apologies for the delay, I am looking into why the CI validation is still failing (it seems to pass for me locally).

In the meantime would you mind verifying the contract with the explorer? 

https://goerli.basescan.org/address/0x52b7D8851d6CcBC6342ba0855Be65f7B82A3F17f#code

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* master
  * **PR #401** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/401" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/401?utm_source=stack-comment).

**Comment by kr-pratikp** [comment]
> Apologies for the delay, I am looking into why the CI validation is still failing (it seems to pass for me locally).
> 
> In the meantime would you mind verifying the contract with the explorer?
> 
> https://goerli.basescan.org/address/0x52b7D8851d6CcBC6342ba0855Be65f7B82A3F17f#code

Verified and Published.

File: data/CHAIN/data.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Chain Token",
```
[the remark this answers is not in the document]
Comment (kr-pratikp) [reply data/CHAIN/data.json]: Updated.

**Review by roberto-bayardo — APPROVED** [review]
Looks good from the Base side.

**Review by nickbalestra — APPROVED** [review]

**Pull request merged**